api/src/basic.rs

```diff
@@ -60,6 +60,47 @@ impl BasicApi {
         Html(OPEN_API_HTML.to_string())
     }
 
+    /// Show some basic info of the node.
+    #[oai(
+        path = "/info",
+        method = "get",
+        operation_id = "info",
+        tag = "ApiTags::General"
+    )]
+    async fn info(&self) -> Html<String> {
+        let mut rows = vec![];
+        rows.push(format!(
+            "<p> Bootstrapping Mode: {:?} </p>",
+            self.context
+                .node_config
+                .state_sync
+                .state_sync_driver
+                .bootstrapping_mode
+        ));
+        rows.push(format!(
+            "<p> Continuous Syncing Mode: {:?} </p>",
+            self.context
+                .node_config
+                .state_sync
+                .state_sync_driver
+                .continuous_syncing_mode
+        ));
+        rows.push(format!(
+            "<p> New Storage Format: {:?} </p>",
+            self.context
+                .node_config
+                .storage
+                .rocksdb_configs
+                .enable_storage_sharding
+        ));
+        rows.push(format!(
+            "<p> Internal Indexer Config: {:?} </p>",
+            self.context.node_config.indexer_db_config
+        ));
+
+        Html(rows.join("\n"))
```

Comment:
Any reason not to send outputs as a map?  It is JSON

Reply:
I did whatever chatgpt told me to do.

Changed to use map now.